test/unit/services/business/fetch-business-details-service.test.js

```diff
@@ -1,13 +1,50 @@
 // Test framework dependencies
-import { describe, test, expect } from 'vitest'
+import { describe, test, expect, beforeEach, vi } from 'vitest'
 
 // Thing under test
 import { fetchBusinessDetailsService } from '../../../../src/services/business/fetch-business-details-service.js'
 
 describe('fetchBusinessDetailsService', () => {
+  let data
+  let yar
+
+  beforeEach(() => {
+    vi.clearAllMocks()
+
+    data = {
```

Comment:
This `data` object is used across tests - should we have a mock object to reuse?